app/controllers/article_controller.rb

```diff
@@ -91,4 +83,18 @@ def search_service
   def set_search_query_modifier
     @search_modifier ||= SearchQueryModifier.new(params, blacklight_config)
   end
+
+  # Reuse the EDS session token if available in the user's session data,
+  # otherwise establish a session
+  def setup_eds_session(session)
+    return if session['eds_session_token'].present?
+    session['eds_session_token'] = EBSCO::EDS::Session.new(
+      guest: session['guest'],
```

Comment:
Hmm, this has me thinking a bit.  The only time that the `guest` status is going to change is when the user logs in or out.  At log-in time, we'll need to determine if the user should be in guest mode or not.  I guess I'm just a bit worried about storing the `guest` status in the session w/o a good handle on how that session variable might get set/unset.

What would you think about just hard-coding this to be `true`, removing the `guest` session variable, and hammering out the implementation details when we need to implement non-guest sessions?